ChurchCRM/CRM
# Have GetAge() return -1 if Year is null, and fix return type

# Description & Issue number it closes 
Closes #6887 - Adds return -1 to GetAge() if Year is null

## Screenshots (if appropriate)
N/A

## How to test the changes?
Perform CSVImport on a small file, with some records having birthdates, and others missing.  Be sure to map the correct field to the birthdate field.  It should fail before this change, and import successfully after this change.

## Type of change


# How Has This Been Tested?
Tested as described above, with simple 2-record csv file.

# Checklist:
